rholang/main/runner.py

```
@@ -35,6 +36,9 @@ class RunError(UserError):
     pass
 
 
+class SubprocessTimeoutError(SubprocessError):
```

This should subclass `UserError` right? The point is: we should report this error to the user. It's not like an http 5xx thing where it's the server's fault.